.github/workflows/deploy-prod-ecs.yml

```diff
@@ -17,13 +20,11 @@ jobs:
       - uses: mbta/actions/build-push-ecr@v1
         id: build-push
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: ${{ secrets.AWS_PROD_ROLE_ARN }}
```

Comment:
Question: Are we setting up different roles for `prod` and `dev` deployment?

Reply:
I don't think we need separate roles.

Reply:
I will use same ARN for both **prod** and **dev**